nf_core/pipelines/schema.py

```diff
@@ -645,6 +645,9 @@ def markdown_param_table(self, properties, required, columns):
                     out += f"| `{p_key}` "
                 elif column == "description":
                     desc = param.get("description", "").replace("\n", "<br>")
+                    if "enum" in param:
+                        enum_values = "\\|".join(f"`{e}`" for e in param["enum"])
+                        desc += f" (Allowed values: {enum_values})"
```

Comment:
Let's use "accepted" to match the output from the `--help` command
```suggestion
                        desc += f" (accepted: {enum_values})"
```

Reply:
I was thinking that too, I'll accept